compose.yml

```diff
@@ -18,6 +18,32 @@ services:
       - ../NextcloudExtract:/var/www/html/custom_apps/extract:rw
     depends_on:
       - db
+      - storage
+
+  storage:
+    image: minio/minio:latest
+    ports:
+      - 9100:9000
+      - 9101:9001
+    env_file:
+      - .env
+    command: 'server /data --console-address ":9001"'
+    volumes:
+      - /var/data/minio:/data
+
+  createbuckets:
```

Comment:
OBJECTSTORE_S3_AUTOCREATE with true does not work. So here the local workaround for it. So we dont have to create the bucket in minio before nextcloud installation